game/ui/commons/options/options.gd

```diff
@@ -7,12 +7,12 @@ onready var backup_settings
 
 
 func _ready():
-	initialize_options(escoria.settings)
+	pass
 
 
 func show():
 	backup_settings = escoria.settings.duplicate()
```

Comment:
Can't this line be removed now?

Reply:
I think this line is necessary to initialize `backup_settings` variable.

Reply:
Oh, riiight. It's a class variable. I missed that.